### ISSUE TITLE
Qt5 SQL Errors

### ISSUE THREAD

**Issue by paddatrapper** [ElvishArtisan/rivendell#675]
When editing a service in RDAdmin using the qt5 build (02ae1d), I get the following error:

```
DB connection re-established
invalid SQL or failed DB connection[You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'GROUPS order by NAME' at line 1 QMYSQL: Unable to execute query]: select NAME from GROUPS order by NAME
```

The error occurs when you open the service to edit it. The DB is initialized using `sudo rddbmgr --create --generate-audio` with nothing else added

**Comment by ElvishArtisan** [comment]
Confirmed here.

**Comment by ElvishArtisan** [comment]
Fixed in ede949fa.
